mullvad-types/Cargo.toml

```diff
@@ -26,3 +26,6 @@ clap = { workspace = true , optional = true }
 
 [target.'cfg(target_os = "android")'.dependencies]
 jnix = { version = "0.5.1", features = ["derive"] }
+
+[dev-dependencies]
+proptest = "1.4"
```

Comment:
We now have three seperate instances of proptest in toml files. Should we make it a workspace dependency?

Reply:
Sure :blush: